### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by zenddignite**
Test PR process with new line in readme

### CODE REVIEW

File: ui/plugin-lifecycle/src/main/services/organisation.service.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    public watchOrgs(): Observable<Organisation[]> {
+        return this._orgsSubject.asObservable();
+    }
+
+    public refresh() {
+        this.loadOrgs();
+    }
+
+    private loadOrgs(): void {
+        this.getAllOrganisations("https://bos1-vcd-sp-static-200-117.eng.vmware.com", this.authService.getAuthToken())
```
Comment (jmoroski) [inline ui/plugin-lifecycle/src/main/services/organisation.service.ts]: Another place where you'll need to rely on the injected container hooks.  See the latest seed project for example usage.

File: ui/plugin-lifecycle/src/main/interfaces/Plugin.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+export interface Plugin {
```
Comment (jmoroski) [inline ui/plugin-lifecycle/src/main/interfaces/Plugin.ts]: many of these objects should also be available in @vcd/bindings...PluginUploadSpec, UiPluginMetadata, etc.

File: ui/api-client/.classpath [hunk 3]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (zenddignite) [reply ui/api-client/.classpath]: I added this files and folders in my .gitignore but they're still not ignored for some reason.

File: ui/plugin-lifecycle/src/main/services/auth.service.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+interface PluginUpdateOptions {
+    tenant_scoped: boolean,
+    provider_scoped: boolean,
+    enabled: boolean
+}
+
+@Injectable()
+export class AuthService {
+    private _baseUrl = "https://bos1-vcd-sp-static-200-117.eng.vmware.com";
```
[the remark this answers is not in the document]
Comment (zenddignite) [reply ui/plugin-lifecycle/src/main/services/auth.service.ts]: Yeah thank you I'll remove that.

File: ui/plugin-lifecycle/src/main/services/auth.service.ts [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        const authString: String = btoa(`${username}@${tenant}:${password}`);
+        const headers = new Headers();
+        headers.append("Authorization", `Basic ${authString}`);
+        headers.append("Accept", "application/*+xml;version=30.0");
+        const opts = new RequestOptions();
+        opts.headers = headers;
+        return this.http.post(`${this._baseUrl}/api/sessions`, null, opts);
+    }
+
+    public auth(): Promise<string> {
```
[the remark this answers is not in the document]
Comment (zenddignite) [reply ui/plugin-lifecycle/src/main/services/auth.service.ts]: The auth.service is replaced with AuthTokenHolderService in plugin-lifecycle branch.

File: ui/plugin-lifecycle/src/main/services/organisation.service.ts (hunk 1 above)
Comment (zenddignite) [reply ui/plugin-lifecycle/src/main/services/organisation.service.ts]: This is already replaced but in plugin-lifecycle branch.

File: ui/plugin-lifecycle/src/main/interfaces/Organisation.ts [hunk 6]
```diff
@@ -0,0 +1,16 @@
+export interface Organisation {
```
[the remark this answers is not in the document]
Comment (zenddignite) [reply ui/plugin-lifecycle/src/main/interfaces/Organisation.ts]: I can't find any class or interface with this name.

File: ui/plugin-lifecycle/src/main/interfaces/Plugin.ts (hunk 2 above)
Comment (zenddignite) [reply ui/plugin-lifecycle/src/main/interfaces/Plugin.ts]: I can't find any class or interface with this names.

Comment (zenddignite) [reply ui/plugin-lifecycle/src/main/interfaces/Plugin.ts]: I had rebuild the packages folder and it's okey.

**Comment by jmoroski** [comment]
This pull request was a test and not intended to be merged.  It is superseded by Pull Request #33.

**Pull request closed without merging**